polls/tool/src/webapp/templates/votePolls.html

```diff
@@ -108,9 +109,9 @@ <h1 rsf:id="poll-list-title">Polls</h1>
       </td>
     </tr>
   </tbody>
-</table>
+</table></div>

```

Comment:
Nice work! Just a quick suggestion: placing </table> and </div> on separate lines improves readability and maintainability. Could you adjust the formatting?